API.md

```diff
@@ -243,3 +243,7 @@ The method returns a `Popen` object that can be used to communicate to the under
 #### `ip_address`
 
 The IP address of the node.
+
+#### `ssh_key_path`
```

Comment:
This documentation is for the public interface of a node. There is no `ssh_key_path` attribute. There is only a `_ ssh_key_path` and private interfaces are not documented.

Reply:
That is - the preferred action is to remove this block.

Reply:
Ok, will remove.

Reply:
Also, FWIW, this would fail CI because of the table of contents. Unfortunately there is currently no way for `make lint` to verify that the ToC is correct locally, only on Travis (see the `Makefile` for details). I think one has to `make lint` after modifying any headings.